server/api/tso.go

```diff
@@ -75,3 +75,46 @@ func (h *tsoHandler) TransferLocalTSOAllocator(w http.ResponseWriter, r *http.Re
 	}
 	h.rd.JSON(w, http.StatusOK, "The transfer command is submitted.")
 }
+
+// @Tags tso
+// @Summary Update dc location of a PD member, mainly for test usage.
+// @Accept json
+// @Param name path string true "PD server name"
+// @Param body body object true "json params"
+// @Produce json
+// @Success 200 {string} string "The dc location config is updated."
+// @Failure 400 {string} string "The input is invalid."
+// @Failure 404 {string} string "The member does not exist."
+// @Failure 500 {string} string "PD server failed to proceed the request."
+// @Router /tso/dc-location/{name} [post]
+func (h *tsoHandler) UpdateDCLocationInfo(w http.ResponseWriter, r *http.Request) {
+	members, membersErr := getMembers(h.svr)
+	if membersErr != nil {
+		h.rd.JSON(w, http.StatusInternalServerError, membersErr.Error())
+		return
+	}
+
+	var memberID uint64
+	name := mux.Vars(r)["name"]
+	for _, m := range members.GetMembers() {
+		if m.GetName() == name {
+			memberID = m.GetMemberId()
+			break
+		}
+	}
+	if memberID == 0 {
+		h.rd.JSON(w, http.StatusNotFound, fmt.Sprintf("not found, pd: %s", name))
+		return
+	}
+
+	dcLocation := r.URL.Query().Get("dcLocation")
+	if len(dcLocation) < 1 {
```

Comment:
```suggestion
	if dcLocation == "" {
```

Reply:
Simply copy the code from `TransferLocalTSOAllocator`. 
I just want to keep it the same in one single file.
IMO there is also no different